lab_3_generate_by_ngrams/start.py

```diff
@@ -11,9 +13,24 @@ def main() -> None:
     """
     with open("./assets/Harry_Potter.txt", "r", encoding="utf-8") as text_file:
         text = text_file.read()
-    result = None
+    textprocessor = TextProcessor('_')
+    encoded_text = textprocessor.encode(text)
+    assert isinstance(encoded_text, tuple)
+    decoded_text = textprocessor.decode(encoded_text)
+    result = decoded_text
+    ngramlanguagemodel = NGramLanguageModel(encoded_text[:100], 3)
+    build = ngramlanguagemodel.build()
+    print(build)
+    ngramlanguagemodel_2 = NGramLanguageModel(encoded_text, 7)
+    greedytextgenerator = GreedyTextGenerator(ngramlanguagemodel_2, textprocessor)
+    predicted_text = greedytextgenerator.run(51, 'Vernon')
+    print(predicted_text)
+    beamsearchtextgenerator = BeamSearchTextGenerator(ngramlanguagemodel_2, textprocessor, 7)
+    predicted_text_2 = beamsearchtextgenerator.run('Vernon', 56)
+    print(predicted_text_2)
     assert result
 
 
+
```

Comment:
I do not think you really need this blank line

Reply:
deleted